lib/galaxy/dependencies/dev-requirements.txt

```diff
@@ -90,7 +90,7 @@ jsonpointer==2.1; python_version >= "3.6" and python_full_version < "3.0.0" or p
 jsonschema==3.2.0
 keystoneauth1==4.3.1; python_version >= "3.6"
 kombu==5.0.2; python_version >= "3.6"
-lagom==1.2.1; python_version >= "3.6" and python_version < "4.0"
+lagom==1.3.0; python_version >= "3.6" and python_version < "4.0"
```

Comment:
Was there a error here ? The test failures looked unrelated ?

Reply:
Here's the issue: 
https://github.com/galaxyproject/galaxy/runs/2170061584#step:6:3603